### ISSUE TITLE
Give an option to not only return the data, but also the header

### ISSUE THREAD

**Issue by vionemc** [SteelPangolin/genderize#4]
There are important informations regarding rate limit in the response header stated [here](https://store.genderize.io/documentation#rate-limiting).

```
X-Rate-Limit-Limit: 100000 // The amount of names in the current subscription period
X-Rate-Limit-Remaining: 1398 // The number of names left in the current subscription period
X-Rate-Reset: 13829 // Seconds remaining until a new subscription period starts
```

I am quite new in Github, I want to contribute here, but how to.

**Comment by vionemc** [comment]
I already made the revision. Please provide me access and see my revision. If you like it, you may approve it and we'd procede to a pull request.

**Comment by SteelPangolin** [comment]
Thanks for the patch! As we discussed in email, you can fork this repo and send me a pull request from your fork, and I'll be happy to review it as soon as I can.

**Comment by SteelPangolin** [comment]
I've merged your patch from PR #5 and will close this issue as soon as I've pushed a new version to PyPI.

**Comment by SteelPangolin** [comment]
https://pypi.python.org/pypi/Genderize/0.1.5
